_posts/2021-04-19-Disney.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+author: Paul-Louis Biondi
+excerpt_separator: <!--more-->
+title: Making Disney Celebrity
+---
+
+## Initial Thoughts
+For this project, I chose to go with Option B, looking at a specific era of film and television. I chose to look at Disney Studios specifically, rather than a specific genre, because of the *unique* influence that Disney has held over American youth, popular culture, and the creation of celebrity and stardom in the United States (and the world more widely as well), especially on Gen Z. For the project, I looked specifically at Disney-made film and television shows from 2001 to 2015. Because Disney Studios is so encompassing in the film and television industry, I focused on "Disney Originals" for the movies from this period and exlcuded cartoon and animated movies. For the sake of the project's span, I omitted a some of the smaller Disney Originals from the period 2001-2015, basing it on the reported viewership from each movie's premieres (Disney Originals all premiered on Disney channel rather than movie theaters). Having grown up as a part of the generation that lived with these movies and shows, I was interested in seeing how the network depicted the use of actors in the Disney Studios. In this way, I think the movie/television nodes are less about the actual project than the time period and the studio behind it. My hypothesis was that there would arise two main ideas in the network. Firstly, that the creation of celebrity was key for the development of Disney and Disney sought to create, rather than maintain, stardom. Thus, they would be responsible for the the creation of pop culture, but not be made to ensure it. Secondly, I also wanted to see how interconnected Disney Studios itself actually is. By looking at their actors and their projects, I hoped to see overlap, creating a distinct "community" of Disney actors.
+  
+## The Network
+![Disney Network](https://github.com/HUM-331-Princeton/HUM-331-Princeton.github.io/blob/8f0d3c559d08a8dbcfb9c370fff68927223881bc/images/Cytoscape%20Disney%20Network.png)
+<!--more-->
+![Disney Network 2](https://github.com/HUM-331-Princeton/HUM-331-Princeton.github.io/blob/9d5285d5f4bff420934b05189be311f0d386f7d9/images/Cytoscape%20Disney%20Partial%20Network%201.png)
+
+![Disney Network 3](https://github.com/HUM-331-Princeton/HUM-331-Princeton.github.io/blob/9d5285d5f4bff420934b05189be311f0d386f7d9/images/Cytoscape%20Disney%20Partial%20Network%202.png)
+
+You can view the Google Sheet I used to create the Cytoscape network [here](https://docs.google.com/spreadsheets/d/1TbMeS3ywwoJoHnECsWKLkGhLXs2MVqlPI1LUds8ct0A/edit#gid=0)
```

Comment:
I followed the link and was denied access! Be sure to share your file and change the settings so that all people can view it.